gradle.properties

```diff
@@ -19,7 +19,7 @@
 #
 
 rootProject.name=fdb-record-layer
-version=4.2.6.0
+version=4.3.0.0
```

Comment:
Not that I'm opposed to it, but was there anything in particular that made it seem like this was a good candidate for the 4.3 bump? The new format version?

Reply:
Yes, I bumped the minor version because there is a new format version. I can undo it.
I forgot to add a comment to the PR, but I now remember that I did this in part to make sure to start a discussion.

Reply:
Discussed offline, and decided that it is worth bumping the minor version in this instance.